provisioning/src/handlers/portfolioDrafts/post.ts

```diff
@@ -0,0 +1,31 @@
+import { Context, HttpRequest } from '@azure/functions'
+const { CosmosClient } = require("@azure/cosmos");
+import { v4 as uuidv4 } from 'uuid';
+import dayjs from "dayjs";
+import {PortfolioSummary} from "../../lib/models/PortfolioSummary";
+import {ProvisioningStatus} from "../../lib/models/ProvisioningStatus";
+
+export async function handler (context: Context, req: HttpRequest): Promise<void> {
+    const connectionString = process.env["COSMOS-CONNECTION-STRING"];
+    context.log("Connecting to " + connectionString)
+    const client = new CosmosClient(connectionString);
+
+    // TODO: create these in terraform & fail fast
```

Comment:
Are these TODOs still active?

Reply:
That particular one is half-active I guess. The DB and Container are in Terraform, but we do still want to fail fast if this code executes and the underlying resources aren't there.